README.md

```diff
@@ -11,15 +11,18 @@ kCTF is a Kubernetes-based infrastructure for CTF competitions.
 
 ## Getting Started
 
-If you want to quickly get started, **follow the [local testing](docs/local-testing.md) walkthrough**, this will show you how to build and test challenges locally.
+The following documentation resources are available to help you get started:
 
-If you want to read more about how kCTF works, take a look at [kCTF in 8 minutes](docs/introduction.md): A quick 8 minutes summary of what is kCTF and how it interacts with Kubernetes.
-
-Once you have everything up and running, you can try deploying to Google Cloud. Follow the [Google Cloud walkthrough](docs/google-cloud.md).
-
-You can also find a [troubleshooting playbook](docs/troubleshooting.md) (for fixing broken challenges), and a guide for [setting up DNS](docs/dns.md) for the CTF.
+* [Local Testing Walkthrough](local-testing.md) – A quick start guide showing you how to build and test challenges locally.
```

Comment:
maybe we can suggest readers a document to start explicitly?

Reply:
@sirdarckcat I updated the branch with an improved version. Does that work better? Thanks!